src/state/shellscripts/env_hook.sh

```diff
@@ -1,6 +1,6 @@
 _opam_env_hook() {
  local previous_exit_status=$?;
- eval $(opam env --shell=bash --readonly);
```

Comment:
errr... shouldn't the redirect be _outside_ of the `$( )` ? :confused:

Reply:
Redirection must be inside `$( )`, but only stderr must be redirected

Reply:
ah yes, indeed the redirect doesn't apply to the shell expansion, which is the part writing to stderr. Good.